src/PaletteForm/PaletteForm.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import './PaletteForm.scss';
+
+class PaletteForm extends Component {
+    constructor() {
+        super()
+        this.state = {
+
+        }
+    }
```

Comment:
We may not need class since we are using hooks, but we can change that as needed